port/common/omrcuda.cpp

```diff
@@ -995,29 +995,29 @@ const J9CudaLibraryDescriptor runtimeLibraries[] = {
 /*
  * Include forward-compatible support for runtime libraries.
  */
-#if 8000 <= CUDA_VERSION
+#if CUDA_VERSION <= 8000
```

Comment:
No, that's what is used to do, but it's not what we wanted. If you build with CUDA 7.0, the 7.x and 8.x runtime libraries are compatible with the API in the 7.0 header files. The reverse is not true: you can't compile with 8.0 header files and expect to succeed using a 7.x runtime.